packages/NodeTypeResolver/PHPStan/TypeHasher.php

```diff
@@ -30,7 +30,7 @@ public function areTypesEqual(Type $firstType, Type $secondType): bool
     public function createTypeHash(Type $type): string
     {
         if ($type instanceof MixedType) {
-            return serialize($type) . $type->isExplicitMixed();
+            return $type->describe(VerbosityLevel::precise()) . $type->isExplicitMixed();
```

Comment:
In some wierd cases, the `MixedType` contains closures that cannot be serialized.